lib/clientv2/networkpolicy_e2e_test.go

```diff
@@ -177,23 +174,27 @@ var _ = testutils.E2eDatastoreDescribe("NetworkPolicy tests", testutils.Datastor
 			Expect(outError.Error()).To(Equal("update conflict: NetworkPolicy(" + namespace1 + "/" + name1 + ")"))
 			Expect(res1.ResourceVersion).To(Equal(rv1_2))
 
-			By("Getting NetworkPolicy (name1) with the original resource version and comparing the output against spec1")
-			res, outError = c.NetworkPolicies().Get(ctx, namespace1, name1, options.GetOptions{ResourceVersion: rv1_1})
-			Expect(outError).NotTo(HaveOccurred())
-			testutils.ExpectResource(res, apiv2.KindNetworkPolicy, namespace1, name1, spec1)
-			Expect(res.ResourceVersion).To(Equal(rv1_1))
+			if config.Spec.DatastoreType != apiconfig.Kubernetes {
```

Comment:
What changed here to make this test etcd only?

Reply:
ah yes - i'll see which tests can be re-instated now.   i had to disable a bunch while kdd was dead.

Reply:
oh this isn't my PR.  lol

nonetheless i should be able to expose a bunch of tests again now.

Reply:
So, the two types of tests that I've disabled are:

1. Getting a resource by a particular revision/resource version. This test is looking up a particular key + revision and expecting the version of the key prior to the update. I've looked but couldn't find a way to do this with crd/k8s rest apis..

2. Specifying a TTL. These tests create a resource with a TTL and expect the resource to be deleted within that time. Storage layer of k8s use this TTL feature but is not exposed over the REST API. I haven't figured out a way to do this yet and tracking this as a separate issue.

Reply:
Is there a GitHub issue tracking this lack of feature parity? If not could you raise one? Thanks!

Reply:
i wonder if there is an annotation for ttl - I know I spotted something in the node yaml that we use in our kdd tests that set a ttl to 0.

Reply:
not suggesting we worry about it at the mo though

Reply:
I'll add a note to look into this as part of feature parity work.